src/sentry/services/hybrid_cloud/app/model.py

```diff
@@ -99,18 +99,23 @@ class SentryAppEventDataInterface(Protocol):
     the minimum required properties.
     """
 
-    id: str
-    label: str
+    @property
+    def id(self) -> str:
+        ...
+
+    @property
+    def label(self) -> str:
+        ...
 
     @property
     def actionType(self) -> str:
-        pass
+        ...
 
     def is_enabled(self) -> bool:
-        pass
+        ...
 
 
-class RpcSentryAppEventData(RpcModel, SentryAppEventDataInterface, metaclass=RpcModelProtocolMeta):
+class RpcSentryAppEventData(RpcModel, metaclass=RpcModelProtocolMeta):
```

Comment:
this is a subtle-but-annoying incompatibility between pydantic and mypy

the protocol as originally written has `id` and `label` as mutable instance variables -- but they're actually immutable class variables.  mixing in the protocol causes pydantic to throw some inscrutable error about metaclass mismatch

fortunately you don't need to inherit from a Protocol -- typing compatibilty for them is implicit